code/_core/obj/item/clothing/neck/goblin_necklace.dm

```diff
@@ -0,0 +1,40 @@
+/obj/item/clothing/neck/goblin_necklace
+	name = "Goblin King's necklace"
+	icon = 'icons/obj/item/clothing/neck/goblin_necklace.dmi'
+	desc = "You monster, stealing from the little green men."
+	desc_extended = "The loin is imbued with the life essence of the late Goblin King. Greatly desired by mages."
+	worn_layer = LAYER_MOB_CLOTHING_NECK_UNDER
+
+	protected_limbs = list(BODY_TORSO)
+
+	rarity = RARITY_LEGENDARY
+
+	defense_rating = list(
+		ARCANE = AP_GREATSWORD,
+		HOLY = -AP_GREATSWORD,
+		DARK = AP_GREATSWORD
+	)
+
+	size = SIZE_1
+
+
+	value = 1000
+
+	var/cooldown_effect = 0
+
+/obj/item/clothing/neck/goblin_necklace/Finalize()
+	. = ..()
+	start_thinking(src)
+
+/obj/item/clothing/neck/goblin_necklace/think()
```

Comment:
I think it would be better if the start_thinking(src) was only called when the loincloth is worn. I'm imagining a scenario where someone has near infinite mana regen by putting 10 pairs of goblin loins in their bluespace backpack.